src/components/jurisdiction/Jurisdiction.stories.ts

```diff
@@ -16,3 +16,14 @@ export const Default = Template.bind({})
 Default['args'] = {
   showUsaJurisdictions: false
 }
+
+export const JurisdictionInitialValue = Template.bind({})
+JurisdictionInitialValue['args'] = {
+  showUsaJurisdictions: false,
+  initialValue: {
```

Comment:
Wait, so I see below the type of this prop a string while here it's an object?

Reply:
looking into it....